csaf/src/verification/check/mod.rs

```diff
@@ -43,6 +46,195 @@ impl Checking {
     }
 }
 
+pub fn check_vulnerabilities_size(csaf: &Csaf) -> Vec<CheckError> {
```

Comment:
I think it would be helpful to have some comments for the `check_` functions, explaining which part of the spec they check.